misc/windows-deploy/amazon-ecs-agent.ps1

```diff
@@ -79,18 +91,18 @@ try {
             docker build -t amazon/amazon-ecs-credential-proxy --file .\credentialproxy.dockerfile . | out-null
             docker run --name ecs-cred-proxy -d -p 80:51679 amazon/amazon-ecs-credential-proxy | out-null
         } catch {
-            LogMsg -message "Running Credential Proxy container failed" -logLevel "INFO"
+            LogMsg -message "Running Credential Proxy container failed." -logLevel "ERROR"
```

Comment:
Similar to the retry logic in loopback.ps1, should we also add retry logic here in case the docker run on the credential proxy fails for some reason ?